digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/dao/linearasset/ManoeuvreDaoSpec.scala

```diff
@@ -21,7 +20,7 @@ class ManoeuvreDaoSpec extends  FunSuite with Matchers {
     runWithRollback {
       val (linkId1, linkId2, linkId3) =
         (LinkIdGenerator.generateRandom(), LinkIdGenerator.generateRandom(), LinkIdGenerator.generateRandom())
-      val dao = new ManoeuvreDao(MockitoSugar.mock[RoadLinkClient])
+      val dao = new ManoeuvreDao()
```

Comment:
Näissä ei varmaan tarvis luoda joka testille daoa uudestaan, mut eipä tuo kovin vaarallista oo.

Reply:
joo en nyt lähtenyt käymään noita läpi, mut joo ei ole tarvetta luoda uudestaan, onhan tossa tietty se että jos noissa object on jotakin sisäisiä arvoja ne säilyy jos käytetään samaa objectia uudestaan. Pitäis laittaa sellaiseen before kohtaan noiden luonti  niin jokainen testi loisi noi aina uudestaan.